tests/app/config/packages/neusta_converter.yaml

```diff
@@ -0,0 +1,16 @@
+neusta_converter:
+  converter:
```

Comment:
Is this duplication of labels necessary?
Or what do you have in mind in parallel to the section "converter"?